### DISCUSSION TITLE
wstunnel_arm_cortex-a7.ipk for openwrt router

### DISCUSSION THREAD

**Discussion by raTHiopA** [erebe/wstunnel#117] [General]
Hi, is it possible to get that binary for my openwrt router?

**Comment by sihiver** [comment]
try wstunnel v3.1, at me work

**Comment by erebe** [comment]
Hello @raTHiopA,

You can try like @sihiver suggest older version, I think somebody build a misp binary at some points.
Regarding providing the binary myself, I don't provide binaries for architecture/devices I don't own myself, as it takes me too much time to do so.

You can look at the build script for raspbian https://github.com/erebe/wstunnel/blob/master/build_raspbian.sh if you want to give it a try for yourself. I you manage to build the binary, let me know I will put it in the release :+1:
